consensus/ibft/ibft.go

```diff
@@ -563,6 +696,19 @@ func (i *Ibft) runSyncState() {
 
 var defaultBlockPeriod = 2 * time.Second
 
+// shouldWriteTransactions checks if each consensus mechanism accepts a block with transactions at given height
+// returns true if all mechanisms accepts
```

Comment:
Typo here, should be `if all mechanisms accept`

Reply:
Fixed at 4dd5210951014804bcc12ecbb507d80374ba1121